front/src/components/templates/MainTemplate/MainTemplate.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Menu from '../../organisms/Menu/Menu';
+import { MainTemplateProps } from '../../../interfaces/templates/MainTemplate/IMainTemplate';
+
+const useStyles = makeStyles(() => createStyles({
+  root: {
+    padding: '60px 64px 32px 64px',
+    backgroundColor: '#f5f5f5',
+    minWidth: '100%',
+    minHeight: '100%',
+    marginTop: 0,
+    marginBottom: 0,
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    overflow: 'hidden',
+  },
+  paper: {
+    padding: '16px',
+    minHeight: '100%',
+    overflow: 'scroll',
```

Comment:
```suggestion
    overflowY: 'scroll',
```
로 하면 더 깔끔해보이는데 어떤가요??

Reply:
맞네요! 수정하겠습니다!